src/Authentication/OldStyleTokens/AccessTokenUrlAuthenticator.php

```diff
@@ -26,7 +26,7 @@ public function addAuthenticationTokensToApplicationUrl( string $url ): string {
 	 * @param class-string $urlGeneratorClass
 	 * @param string[] $requestedParameters
 	 *
-	 * @return array<string, int|string>
+	 * @return array<string, bool|int|string>
```

Comment:
The boolean comes from the `json_encode` and is confusing/misleading. Please add `JSON_THROW_ON_ERROR` instead to ensure `custom` is a always string